src/core/icons/fileIcons.ts

```diff
@@ -1,201 +1,201 @@
-import type { FileIcons } from '../models/icons/files/fileTypes';
-import { IconPack } from '../models/icons/iconPack';
-import { FileNamePattern } from '../models/icons/patterns/patterns';
-import { parseByPattern } from '../patterns/patterns';
+import type { FileIcons } from "../models/icons/files/fileTypes";
```

Comment:
Please keep the formatting of this file as it was. We already have formatting rules in place so there's no need to update it here.

Reply:
@PKief Changed Formatting -> https://github.com/material-extensions/vscode-material-icon-theme/pull/2573/commits/eb549e6e6af38075ab86cc9d069b578e74b6b9dc